tests/common/platform/processes_utils.py

```diff
@@ -57,12 +57,13 @@ def check_critical_processes(dut, watch_secs=0):
         watch_secs = watch_secs - 5
 
 
-def wait_critical_processes(dut):
+def wait_critical_processes(dut, timeout=None):
```

Comment:
Is it possible to identify the DUT platform from variable `dut`? If set the timeout to 600s for E1031 in this function, then it'll work everywhere.